rj_gameplay/classifier/pass_success_classifier.py

```diff
@@ -0,0 +1,95 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+
+import numpy as np
+import math as m
+
+def classifier(world_state: rc.WorldState) -> bool:
+    our_robots = world_state.our_robots
+    opp_robots = world_state.their_robots
+    ball = world_state.ball
+
+    # check if any robot on our team has the ball 
+    for robot in our_robots:
+        if robot.has_ball:
+            return True
+
+    for robot in opp_robots:
+        if robot.has_ball:
+            return False
```

Comment:
So for the classifiers, we actually won't have access to things like robot.has_ball (this is never true for opponent robots, since we don't have access to their statuses). Instead of using has_ball  try using something like a robots distance to the ball. You can check Yudai's PR, he did something similar.